proxy_test.go

```diff
@@ -89,16 +221,22 @@ upstreams:
 			if err != nil {
 				t.Fatal(err)
 			}
-			rec := httptest.NewRecorder()
-			http.DefaultServeMux.ServeHTTP(rec, req)
-			if rec.Body.String() != test.body {
-				t.Errorf("Expected body %s, got %s", test.body, rec.Body.String())
+			res, err := http.DefaultClient.Do(req)
```

Comment:
Since gondola is started with the Run method, we will actually send a request to gondola, which is a proxy, and verify req/res with upstreams.